R/analyze_binary.R

```diff
@@ -29,7 +29,7 @@ deepstate_harness_analyze_pkg <- function(path, testfiles="all", max_inputs="all
       test.files <- test.files[1:testfiles]
     }
     for(pkg.i in seq_along(test.files)){
-      list_testfiles[[basename(test.files[[pkg.i]])]] <- deepstate_analyze_fun(path,basename(test.files[[pkg.i]]), max_inputs, verbose)
+      list_testfiles[[basename(test.files[[pkg.i]])]] <- deepstate_analyze_fun(path,basename(test.files[[pkg.i]]), max_inputs, verbose=verbose)
```

Comment:
since you use `basename(test.files[[pkg.i]])` twice please create an intermediate variable

Reply:
Done! Update in the commit 7b4d7e9